src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -422,6 +426,21 @@ export class OsmosisSwapper implements Swapper<ChainId> {
         gas: ibcFromOsmosisFeeData.fast.chainSpecific.gasLimit,
         feeDenom: 'uosmo',
       })
+
+      // wait till confirmed
+      const pollResult = await pollForCrossChainComplete({
+        txid: tradeId,
+        baseUrl: osmoUrl,
+      })
```

Comment:
We were never actually polling for OSMO-side Tx for ATOM -> OSMO but rather being optimistic, which is obviously no bueno